position/async/position.js

```diff
@@ -1,34 +1,42 @@
+const sort = require('ssb-sort')
 const pull = require('pull-stream')
 const pullAsync = require('pull-async')
-const GetPoll = require('../../poll/async/get')
-const sort = require('ssb-sort')
 // var { link } = require('ssb-msg-schemas/util')
-//
+
+const isPosition = require('../../isPosition')
+// const GetPoll = require('../../poll/async/get') // ??!!! DOESN't WORK
```

Comment:
The problem is near here.
You log out here and you get nonsense like `{}` instead of a function

Commenting this out and moving it inside the functions fixes things.
The error gets thrown when the inject is run. I've put a method checker in the inject function (see the `assert`s) and they check out ...